docs/docs/cmd/spo/navigation/navigation-node-get.md

```diff
@@ -0,0 +1,88 @@
+# spo navigation node get
+
+Gets information about a specific navigation node.
+
+## Usage
+
+```sh
+m365 spo navigation node get [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: Absolute URL of the site.
+
+`--id <id>`
+: Id of the navigation node.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Retrieve information for a specific navigation node.
+
+```sh
+m365 spo navigation node get --webUrl https://contoso.sharepoint.com/sites/team-a --id 2209
+```
+
+## Response
+
+=== "JSON"
+
+    ```json
+    {
+      "AudienceIds": null,
```

Comment:
I wonder if it would be worth adding some `AudienceIds` so that the user will now what to expect  🤔
what do you think? Basically what we will have here is list of GUIDs for each group that has permission to view this node like 
```
"AudienceIds": [
    "0d718612-8407-4d6b-833c-6891a553354f",
    "f864446f-b4d7-475a-a2ba-1080c6474020"
  ],
```